docs/webhooks/securing-your-webhook.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Securing your webhook
+excerpt: Ensure your server only handles requests sent by Fiberplane
+category: TODO
+slug: webhooks-security
+---
+
+After you created your webhook, you received a shared secret which is a random 32 character hexadecimal string.
+If you did not save it, you can update the webhook and request a regeneration of the shared secret.
+
+The request body will be hashed signed using that shared secret using the algorithm **HMAC SHA-512**. The signature
+will be sent with every request in the `X-Fiberplane-Signature` header. The value of the header is `v1=[signature]`.
+
```

Comment:
Since the whole page talks about webhook security in general, I think it's important to add a small paragraph that reminds that this shared secret only protects from forging and tampering with messages; it does _not_ ensure secrecy. If users want secrecy, they should use HTTPS for their delivery handler (with a Mozilla trusted certificate)